core/templates/components/rubrics-editor/rubrics-editor.directive.ts

```diff
@@ -150,12 +152,15 @@ angular.module('oppia').directive('rubricsEditor', [
           if (ContextService.getPageContext() !== 'skill_editor') {
             $scope.$watch(function() {
               return SkillCreationService.getSkillDescriptionStatus();
-            }, function() {
-              var explanations = ctrl.getRubrics()[1].getExplanations();
-              var difficulty = ctrl.getRubrics()[1].getDifficulty();
-              explanationsMemento[difficulty] = angular.copy(explanations);
-              ctrl.editableExplanations[difficulty] = (
-                angular.copy(explanations));
+            }, function(newValue, oldValue) {
+              if (newValue === SKILL_DESCRIPTION_STATUS_VALUES.STATUS_CHANGED) {
+                var explanations = ctrl.getRubrics()[1].getExplanations();
```

Comment:
Same issue ... don't use [1]? (There may be more than one rubric for a given difficulty, right?)

Reply:
This is the place where the first explanation for the 'Medium' difficulty gets autofilled with the skill description and so we have to use [1] here. Should I add a comment?
Just a note that we can add more explanations for 'Medium', just that the first gets autofilled with the skill description only during creation. In the skill editor, this section would not be called.

Reply:
Yep, please add a detailed comment. (In general, if the reviewer has confusion about something, go ahead and add the comment proactively, since other devs will probably be confused too and that will save review cycles. Thanks!)

Reply:
Ok, done.